examples/get-started/pure-js/esri-arcgis/app.js

```diff
@@ -0,0 +1,58 @@
+import ArcGISMap from 'esri/Map';
+import MapView from 'esri/views/MapView';
+import {EsriDeckLayer} from '@deck.gl/arcgis';
+import {GeoJsonLayer, ArcLayer} from '@deck.gl/layers';
+
+// source: Natural Earth http://www.naturalearthdata.com/ via geojson.xyz
+const AIR_PORTS =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_10m_airports.geojson';
+
+const layer = new EsriDeckLayer({
+  getDeckLayer() {
```

Comment:
Is this called on every redraw? Can the user just pass a `layers` array and only update it when needed?